### PR TITLE
ANDAUTH-69 Remove Hardcoded Dimentions and Colors

### PR DESCRIPTION
**Pull request by somenath1435**
Make use of colors.xml and dimens.xml

### CODE REVIEW

**Comment by InsanusMokrassar** [comment]
please, update all pull requests according to updates in source repo

**Comment by somenath1435** [comment]
@InsanusMokrassar I've updated all of my PRs

File: app/src/main/res/layout/act_grant_permission.xml [hunk 1]
```diff
@@ -96,19 +96,19 @@
         android:layout_alignParentBottom="true"
         android:minHeight="54dip"
```
Comment (InsanusMokrassar) [inline app/src/main/res/layout/act_grant_permission.xml]: You left this, please, fix it

File: app/src/main/res/layout/activity_sync_settings.xml [hunk 2]
```diff
@@ -57,54 +57,54 @@
...
+                                android:layout_marginStart="@dimen/marginMedium"
+                                android:layout_marginLeft="@dimen/marginMedium"/>
 
                         <TextView
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
                                 android:id="@+id/tvSelectedSyncAcc"
-                                android:textSize="18sp"
+                                android:textSize="@dimen/textDefaultMediumSize"
                                 android:textColor="#000000"
```
Comment (InsanusMokrassar) [inline app/src/main/res/layout/activity_sync_settings.xml]: One more skipped

File: app/src/main/res/layout/viewflipper_signin.xml [hunk 3]
```diff
@@ -114,7 +114,7 @@
                         android:layout_height="1dp"
```
Comment (InsanusMokrassar) [inline app/src/main/res/layout/viewflipper_signin.xml]: One more hardcoded

**Review by InsanusMokrassar — CHANGES_REQUESTED** [review]
Please, check again all strings, there is some hardcoded, but I can't choose them during review. For example: https://github.com/xwiki-contrib/android-authenticator/pull/48/files#diff-e583f742aaca2eda54730c48f1fcc6f5R20

**Comment by somenath1435** [comment]
@InsanusMokrassar I've made chamges

File: app/src/main/res/layout/viewflipper_signin.xml [hunk 4]
```diff
@@ -146,15 +146,15 @@
...
+                        android:layout_height="@dimen/imageview_36dp"
                         android:layout_gravity="center"
                         android:src="@drawable/logo"/>
 
                 <TextView
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
-                        android:layout_marginTop="8dp"
+                        android:layout_marginTop="@dimen/marginMedium"
                         android:text="OpenID Connect"
```
Comment (InsanusMokrassar) [inline app/src/main/res/layout/viewflipper_signin.xml]: Still hardcoded

File: app/src/main/res/layout/activity_sync_settings.xml [hunk 5]
```diff
@@ -57,54 +57,54 @@
                         <TextView
                                 android:layout_width="wrap_content"
                                 android:layout_height="wrap_content"
-                                android:layout_marginTop="8dp"
+                                android:layout_marginTop="@dimen/marginMedium"
                                 android:id="@+id/syncAccount"
                                 android:text="Sync Account"
```
Comment (InsanusMokrassar) [inline app/src/main/res/layout/activity_sync_settings.xml]: Still hardcoded

File: app/src/main/res/layout/list_item_group.xml [hunk 6]
```diff
@@ -10,15 +10,15 @@
...
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginLeft="6dp"
-            android:layout_marginTop="10dip"
+            android:layout_marginLeft="@dimen/margin_6dp"
+            android:layout_marginTop="@dimen/margin_10dp"
             android:ellipsize="end"
             android:maxLines="1"
             android:gravity="start"
             android:text="Admin"
```
Comment (InsanusMokrassar) [inline app/src/main/res/layout/list_item_group.xml]: Still hardcoded

File: app/src/main/res/layout/list_item_group.xml [hunk 7]
```diff
@@ -27,8 +27,8 @@
             android:layout_height="wrap_content"
             android:layout_alignLeft="@id/groupName"
             android:layout_below="@id/groupName"
-            android:layout_marginBottom="5dp"
-            android:layout_marginTop="5dip"
+            android:layout_marginBottom="@dimen/margin_5dp"
+            android:layout_marginTop="@dimen/margin_5dp"
             android:text="creator:xwiki"
```
Comment (InsanusMokrassar) [inline app/src/main/res/layout/list_item_group.xml]: Still hardcoded

**Review by InsanusMokrassar — CHANGES_REQUESTED** [review]
https://github.com/xwiki-contrib/android-authenticator/pull/48/files#diff-e583f742aaca2eda54730c48f1fcc6f5R126
https://github.com/xwiki-contrib/android-authenticator/pull/48/files#diff-e583f742aaca2eda54730c48f1fcc6f5R22
https://github.com/xwiki-contrib/android-authenticator/pull/48/files#diff-cc2e231ce27ac95bc207fbbf102660bcR82
These lines are still hardcoded as and other in this review

**Comment by somenath1435** [comment]
@InsanusMokrassar I've removed hardcoded strings as well

**Review by InsanusMokrassar — APPROVED** [review]

**Review by InsanusMokrassar — CHANGES_REQUESTED** [review]
Ok. Please recheck again all sources for absent of hardcoded strings/hints/dimens and I will merge it after check that everythin is look like ok

**Comment by InsanusMokrassar** [comment]
@somenath1435 please, notify me, when you will recheck all

**Comment by somenath1435** [comment]
@InsanusMokrassar I've checked , looks like this PR is ready

**Comment by InsanusMokrassar** [comment]
> @InsanusMokrassar I've checked , looks like this PR is ready

Ok, let me check current state

**Pull request merged**